paladin-core/src/common/mod.rs

```diff
@@ -0,0 +1,9 @@
+use rand::{distributions::Alphanumeric, Rng};
+
+pub fn get_random_routing_key() -> String {
+    rand::thread_rng()
+        .sample_iter(&Alphanumeric)
+        .take(10)
```

Comment:
Is there a reason we need this so big?

Reply:
no 
we can reduce it to maybe 5 or something